proto/zetachain/zetacore/observer/params.proto

```diff
@@ -34,6 +34,10 @@ message ChainParams {
 
   // Advanced confirmation parameters for chain to support fast observation
   ConfirmationParams confirmation_params = 18;
+
+  // Skip actions that require scanning the contents of each block.
+  // The main thing this disables is transfers directly to the TSS address.
+  bool disable_block_scan = 19;
```

Comment:
Would disable_tss_inbound , be more appropriate. 
Will we use this param for any other checks in the future?